frontend/src/views/Pages/AccountOverview/GdtTransactionList.vue

```diff
@@ -223,15 +90,6 @@ export default {
           this.$toasted.error(error.message)
         })
     },
-    getIcon(givenType) {
-      const type = iconByType(givenType)
-      if (type)
-        return {
-          icon: type.icon,
-          class: type.classes + ' m-mb-1 font2em',
-        }
-      this.throwError('no icon to given type: ' + givenType)
-    },
     throwError(msg) {
```

Comment:
This is not needed anymore.

Reply:
das muss schon raus sein, ich habe nur noch code bis zeile 122

Reply:
Ist immer noch da. Zeile 93 bis 95